src/fides/api/ops/models/connectionconfig.py

```diff
@@ -125,6 +127,10 @@ class ConnectionConfig(Base):
     saas_config = Column(
         MutableDict.as_mutable(JSONB), index=False, unique=False, nullable=True
     )
+
+    # This is imported here to avoid a circular dependency
+    from fides.api.ctl.sql_models import System  # type: ignore[attr-defined]
+
     system_id = Column(
         String, ForeignKey(System.id_field_path), nullable=True, index=True
```

Comment:
Alternatively, you can define the FK like this:

```
  system_id = Column(
        String, ForeignKey("ctl_systems.id"), nullable=True, index=True
    )
```

and 

```
 system = relationship("System")
```

instead of importing System like this, but what suddenly prompted the circular dependency?

Reply:
some way it was configured before, I don't remember now exactly why it happened